tests/Queue/QueueSizeTest.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Illuminate\Tests\Integration\Queue;
 
+use Queue;
```

Comment:
This needs to be the actual name of the facade please. `Illuminate\Support\Facades\Queue`.

Reply:
okay, I just was inspired by /tests/Integration/Queue/QueuedListenersTest.php ;-)

Reply:
@hlorofos that's an integration test. The facades are available there. This is a unit test so the framework isn't loaded.

Reply:
@driesvints thanks for your explanations!